tests/MessagePack.SourceGenerator.Tests/Resources/OpenNullableFormatter/MessagePack.GeneratedMessagePackResolver.g.cs

```diff
@@ -0,0 +1,65 @@
+// <auto-generated />
+
+#pragma warning disable 618, 612, 414, 168, CS1591, SA1129, SA1309, SA1312, SA1403, SA1649
+
+using MsgPack = global::MessagePack;
+
+[assembly: MsgPack::Internal.GeneratedAssemblyMessagePackResolverAttribute(typeof(MessagePack.GeneratedMessagePackResolver), 3, 0)]
+
+namespace MessagePack {
+
+/// <summary>A MessagePack resolver that uses generated formatters for types in this assembly.</summary>
+partial class GeneratedMessagePackResolver : MsgPack::IFormatterResolver
+{
+	/// <summary>An instance of this resolver that only returns formatters specifically generated for types in this assembly.</summary>
+	public static readonly MsgPack::IFormatterResolver Instance = new GeneratedMessagePackResolver();
+
+	private GeneratedMessagePackResolver()
+	{
+	}
+
+	public MsgPack::Formatters.IMessagePackFormatter<T> GetFormatter<T>()
+	{
+		return FormatterCache<T>.Formatter;
+	}
+
+	private static class FormatterCache<T>
+	{
+		internal static readonly MsgPack::Formatters.IMessagePackFormatter<T> Formatter;
+
+		static FormatterCache()
+		{
+			var f = GeneratedMessagePackResolverGetFormatterHelper.GetFormatter(typeof(T));
+			if (f != null)
+			{
+				Formatter = (MsgPack::Formatters.IMessagePackFormatter<T>)f;
+			}
+		}
+	}
+
+	private static class GeneratedMessagePackResolverGetFormatterHelper
+	{
+		private static readonly global::System.Collections.Generic.Dictionary<global::System.Type, int> openTypeLookup = new(2)
+		{
+			{ typeof(global::System.Nullable<>), 0 },
```

Comment:
In v2, `DynamicGenericResolver.Instance` was not added to `StandardResolverHelper.DefaultResolvers` in Unity IL2CPP, so it couldn't handle `Nullable<>` dynamically. In v3, although most of the source code was moved outside MessagePack.UnityClient, but [the code](https://github.com/MessagePack-CSharp/MessagePack-CSharp/blob/7896a6d4e8ed9744874e22ddd657105e8773501d/src/MessagePack/Resolvers/StandardResolver.cs#L4) is still there. 

If `DynamicGenericResolver.Instance` is supposed to be added to `StandardResolverHelper.DefaultResolvers` in v3 (since it would be used as nupkg in Unity), I think types like `Nullable<>` and other known generic types (`List<>`, `HashSet<>`, ...) are not necessary in `openTypeLookup`. Should I remove these types from `openTypeLookup`?